### DISCUSSION TITLE
Integrating Timezone Scalars into graphql-scalars

### DISCUSSION THREAD

**Discussion by mannyluvstacos** [Urigo/graphql-scalars#1014] [Q&A]
Firstly, if this is an inappropriate request, please pardon! This is not my intention.
Reading the contribution section, I want to acknowledge the individuality of the developers and recognize the importance of their work as they envision it.

I saw that [graphql-timezone](https://github.com/homoky/graphql-timezone) existed and I was curious y'alls thoughts on collaborating with @homoky (also @homoky 's openness to) having those scalars integrated into `graphql-scalars` to consolidate?

:) Maybe the answer is, "if you're interested in a Timezone scalar, Homoky has you set!"
And that would be completely appropriate.

I am curious how folks would collaborate in this scenario.

i look forward to hearing back!

Respectfully,
Manny

**Comment by ardatan** [comment]
We're always open to have PRs for more scalars :)

**Reply by mannyluvstacos** [reply]
Great! I will look into getting a PR following the contribution guidelines, Arda!

Thank you for the guidance!
